src/log.js

```diff
@@ -219,6 +219,7 @@ export class Log {
    * @param {...*} var_args Arguments substituted into %s in the message.
    * @return {T} The value of shouldBeTrueish.
    * @template T
+   * @throws {Error}
```

Comment:
you are right, that is the case. "The type checker does not currently use this information. It is only used to figure out if a function declared in an externs file has side effects"